### PR TITLE
pacific: test: No direct use of nose

### PR DESCRIPTION
**Pull request by tserong**
backport tracker: https://tracker.ceph.com/issues/55616

---

backport of https://github.com/ceph/ceph/pull/45064
parent tracker: https://tracker.ceph.com/issues/54252

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

**Comment by tserong** [comment]
Note to whoever's merging backports these days: I've pulled in #45650 to get the debian/control changes to apply cleanly in this PR. LMK if you want me to rebase after that PR is merged or if you need me to do anything else to assist.

File: debian/control [hunk 1]
```diff
@@ -37,7 +37,7 @@ Build-Depends: automake,
                libcurl4-openssl-dev,
 # Jaeger       libevent-dev,
                libexpat1-dev,
-# Make-Check   libffi-dev [!amd64],
+               libffi-dev [!amd64] <pkg.ceph.check>,
```
Comment (tchaikov) [inline debian/control:40]: might want to drop the line of 

> (cherry picked from commit https://github.com/ceph/ceph/commit/acf9a0b8e95bd8f5412dc0c11f4a5bfc80199104)

in commit message.

Comment (tserong) [reply debian/control:40]: Fixed, thanks.

**Review by tchaikov — APPROVED** [review]

**Comment by ljflores** [comment]
Rados suite review for documentation:

http://pulpito.front.sepia.ceph.com/yuriw-2022-06-07_19:48:58-rados-wip-yuri6-testing-2022-06-07-0955-pacific-distro-default-smithi/

Failures, unrelated:
    1. https://tracker.ceph.com/issues/53939
    2. https://tracker.ceph.com/issues/53827
    3. https://tracker.ceph.com/issues/54071
    4. https://tracker.ceph.com/issues/54992
    5. https://tracker.ceph.com/issues/49525
    6. https://tracker.ceph.com/issues/55741
    7. https://tracker.ceph.com/issues/52321
    8. https://tracker.ceph.com/issues/50222
    9. https://tracker.ceph.com/issues/53501
    10. https://tracker.ceph.com/issues/56028 --> opened a new Tracker for this; it is unrelated to the PRs that were tested in this run.

Details:
    1. ceph-nfs-upgrade, pacific: Upgrade Paused due to UPGRADE_REDEPLOY_DAEMON: Upgrading daemon osd.0 on host smithi103 failed - Ceph - Orchestrator
    2. cephadm exited with error code when creating osd: Input/Output error. Faulty NVME? - Ceph - Orchestrator
    3. rados/cephadm/osds: Invalid command: missing required parameter hostname(<string>) - Ceph - Orchestrator
    4. cannot stat '/etc/containers/registries.conf': No such file or directory - Ceph - Orchstrator
    5. found snap mapper error on pg 3.2s1 oid 3:4abe9991:::smithi10121515-14:e4 snaps missing in mapper, should be: dc was r -2...repaired - Ceph - RADOS (pending Pacific backport)
    6. cephadm/test_dashboard_e2e.sh: Unable to find element `cd-modal .custom-control-label` when testing on orchestrator/01-hosts.e2e-spec.ts - Ceph - Mgr - Dashboard
    7. qa/tasks/rook times out: 'check osd count' reached maximum tries (90) after waiting for 900 seconds - Ceph - Orchestrator
    8. osd: 5.2s0 deep-scrub : stat mismatch - Ceph - RADOS
    9. Exception when running 'rook' task. - Ceph - Orchestrator
    10. thrash_cache_writeback_proxy_none: FAILED ceph_assert(version == old_value.version) in src/test/osd/RadosModel.h - Ceph - RADOS

**Pull request merged**